sandbox/src/PersonalizationFormBased.js

```diff
@@ -13,7 +13,10 @@ const metadata = {
 
 const usePropositions = () => {
   const [propositions, setPropositions] = useState(undefined);
```

Comment:
Same here, I would have a dedicated page for the new `personalization -> surface` functionality. We could deprecate them in a separate PR.

Reply:
As I mentioned in the previous comment reply, we do need to also modify all existing sandbox pages to prioritize the new `personalization -> decisionScopes` format. However, as part of AJO tests, I will also be adding a dedicated sandbox page for AJO

Reply:
Reverted